proposals/0011-inline-spirv.md

```diff
@@ -211,49 +211,54 @@ With the existing inline SPIR-V, the variable can be decorated and assigned to
 the correct storage class. However, it cannot always be correctly associated
 with the entry point.
 
-To support adding built in inputs from source, this proposal adds a new
-`vk::ext_builtin_input` attribute that applies to a function declaration
-providing an implicit definition which returns the value of the builtin.
+To support adding builtin inputs from source, this proposal adds a new
+`vk::ext_builtin_input` attribute which takes a `builtInId` parameter and
+applies to a variable declaration. This attribute must be applied to a static
+variable.
 
 For example, the `gl_NumWorkGroups` builtin could be declared in a header file
 like this:
 
 ```
 [[vk::ext_builtin_input(/* NumWorkgroups */ 24)]]
-uint3 gl_NumWorkGroups();
+static uint3 gl_NumWorkGroups;
```

Comment:
Since we cannot write to the builtin input, do we want to make the variable const as well? Then we can get all of the checks that it cannot be written to for free.

Reply:
Sure, that seems like a good idea. Updated.